packages/nestjs-module/package.json

```diff
@@ -27,7 +27,7 @@
   "peerDependencies": {
     "@nestjs/common": "^9.0.0 || ^10.0.0",
     "@nestjs/core": "^9.0.0 || ^10.0.0",
-    "reflect-metadata": "^0.1.0",
+    "reflect-metadata": "^0.1.0 || ^0.2.1",
```

Comment:
both versions are compatible with each other with no breaking changes (and this was the main reason for me making this pr, to update the peer dep range)